SCHIZO/Creatures/Components/Carryable.cs

```diff
@@ -0,0 +1,118 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using SCHIZO.Helpers;
+using SCHIZO.Resources;
+using SCHIZO.Sounds.Collections;
+using SCHIZO.Sounds.Players;
+using UnityEngine;
+using Random = UnityEngine.Random;
+
+namespace SCHIZO.Creatures.Components;
+
+partial class Carryable
+{
+    public bool isCarried;
+    public Func<Carryable, CarryCreature, bool> CanAttach;
+    public Action<Carryable, CarryCreature> Attached;
+    public Action<Carryable, CarryCreature> Detached;
+
+    private Creature creature;
+
+    private float _nextCarryNoiseTime;
+    private float _lastPickedUpTime;
+    private List<Behaviour> _disabledComponents;
+    private static readonly List<Type> toDisable = new()
+    {
+        typeof(WorldAmbientSoundPlayer),
```

Comment:
are you sure you want to disable ambient sounds for carried creatures?

Reply:
yeah, the tutel has a different (more panicked) set of sounds to play while carried (and i wanted to have the ermfish say "weee" or something idk)

otherwise the sounds double up

Reply:
i guess we can leave it enabled if we make Carryable temporarily replace the sound collection, your call on what's more scuffed

Reply:
> yeah, the tutel has a different (more panicked) set of sounds to play while carried (and i wanted to have the ermfish say "weee" or something idk)
are those implemented yet?

Reply:
tutel yes, ermfish no - we can take a look once the new sounds from recent streams are added